src/api/Feed.js

```diff
@@ -178,10 +181,7 @@ class Feed extends Base {
             this.commentsAPI.offsetGet(
                 this.id,
                 resolve,
-                err => {
-                    this.errorCallback(err);
-                    resolve();
-                },
+                this.fetchFeedItemErrorCallback.bind(this, resolve),
```

Comment:
I think @priyajeet mentioned using arrow functions in a previous comment, but otherwise LGTM.